lib/mpd.nit

```diff
@@ -162,7 +162,7 @@ class MPDConnection
 		var time: nullable Int = null
 
 		socket.write("currentsong\n")
-		var rep = socket.read(1024)
+		var rep = socket.read(1024).to_s
```

Comment:
While I understand that `to_s` make sense to keep compatibility, I am not sure the original code is robust (or even sound). Does replacing the `read(1024)` with `read_line` a better approach? @xymus you are the original author, what is your opinion? The code is old and may predate the `read_line` method.